boards/arm/usb_kw24d512/Makefile.board

```diff
@@ -0,0 +1,7 @@
+FLASH_SCRIPT = openocd.sh
+DEBUG_SCRIPT = openocd.sh
```

Comment:
Does OpenOCD support this part upstream? I don't know offhand. Can you add jlink support?